PowerEditor/src/functionList.xml

```diff
@@ -318,13 +318,13 @@
 			<parser
 				id         ="nsis_syntax"
 				displayName="NSIS Syntax"
-				commentExpr="(?s:/\*.*?\*/)|(?m-s:[#;].*?$)"
+				commentExpr="(?s:^[\t\x20]*/\*.*?\*/)|(?m-s:^[\t\x20]*[#;].*?$)"
```

Comment:
Here's a perfectly valid NSIS script: [Test.nsi.txt](https://github.com/notepad-plus-plus/notepad-plus-plus/files/545170/Test.nsi.txt)
No function is identified with the default parser rules.

With the changes I've suggested (almost) all functions are detected. SubTest4 is of course incorrectly displayed, but still, it becomes much easier to navigate inside large scripts.

Nevertheless, not a critical problem. I'm waiting for the improved parsers...